static/js/contextMenu.js

```diff
@@ -0,0 +1,355 @@
+var selectedLineNumber;
```

Comment:
Enclose these inside a function or module.  So that you avoid accidentally interfering with variable scoping.